categories.txt

```diff
@@ -0,0 +1,28 @@
+Android
```

Comment:
Can we add a comment here where else to update this?

Reply:
The problem with this option is that we have to take this into account when reading the file (it makes the code to read the file a bit more complicated). Or can you think on an easy way to do this?

Reply:
I was thinking about adding a check to the CI that ensure the issue templates and this file are in sync. But I was not sure if the effort is worth as we don't plan to update the categories often. This is not an issues this PR introduces as we have the problem already. What about merging this PR and discuss how to ensure categories are up-to-date in an issue @mr-tz ? (I can take care of opening the issue)

Reply:
Sounds good! Let's keep it simple, but happy to discuss further separately.

Reply:
Issue created: https://github.com/mandiant/VM-Packages/issues/298